pkgs/applications/graphics/ufraw/default.nix

```diff
@@ -13,6 +13,17 @@ stdenv.mkDerivation rec {
     sha256 = "0pm216pg0vr44gwz9vcvq3fsf8r5iayljhf5nis2mnw7wn6d5azp";
   };
 
+  patches = [
```

Comment:
See review at #64353.

Those patches could be improved.

Reply:
Roger that.
@neilmayhew could you test the suggested changes and update the patches at sourceforge.net?

Reply:
@gloaming Done. The URLs are:

* https://sourceforge.net/p/ufraw/bugs/_discuss/thread/c0761df338/a2d4/attachment/0001-Change-use-of-header-files.patch
* https://sourceforge.net/p/ufraw/bugs/_discuss/thread/c0761df338/a2d4/attachment/0002-Use-symbolic-error-code.patch

Not as tidy as before, but I don't think SourceForge provides a way to modify attachments in the issue description. I had to do it by attaching the new files to a comment.

Reply:
Thanks! Sorry I didn't notice your PR before opening this one.

Reply:
> Sorry I didn't notice your PR before opening this one.

No worries. It might have helped if I'd created an issue as well as a PR. I'll make sure to do that in future.